values/src/main/java/org/spine3/time/LocalDates.java

```diff
@@ -48,15 +52,119 @@ public static LocalDate now() {
     }
 
     /**
-     * Obtains current LocalDate instance with the specified number of days added.
+     * Obtains local date from a year, month and day.
+     */
+    public static LocalDate of(int year, MonthOfYear month, int day) {
+        checkPositive(year, "year");
+        checkPositive(day, "day");
+        final LocalDate result = LocalDate.newBuilder()
+                                          .setYear(year)
+                                          .setMonth(month)
+                                          .setDay(day)
+                                          .build();
+        return result;
+    }
+
+    /**
+     * Obtains a copy of this local date with the specified number of years added.
+     */
+    public static LocalDate plusYears(LocalDate localDate, int yearsToAdd) {
+        checkPositive(yearsToAdd, "yearsToAdd");
+        return changeYear(localDate, yearsToAdd);
+    }
+
+    /**
+     * Obtains a copy of this local date with the specified number of months added.
      */
-    public static LocalDate plusDays(int daysToAdd) {
+    public static LocalDate plusMonths(LocalDate localDate, int monthsToAdd) {
+        checkPositive(monthsToAdd, "monthsToAdd");
+        return changeMonth(localDate, monthsToAdd);
+    }
+
+    /**
+     * Obtains a copy of this local date with the specified number of days added.
+     */
+    public static LocalDate plusDays(LocalDate localDate, int daysToAdd) {
+        checkPositive(daysToAdd, "daysToAdd");
+        return changeDays(localDate, daysToAdd);
+    }
+
+    /**
+     * Obtains a copy of this local date with the specified number of years subtracted.
+     */
+    public static LocalDate minusYears(LocalDate localDate, int yearsToSubtract) {
+        checkPositive(yearsToSubtract, "yearsToSubtract");
+        return changeYear(localDate, -yearsToSubtract);
+    }
+
+    /**
+     * Obtains a copy of this local date with the specified number of months subtracted.
+     */
+    public static LocalDate minusMonths(LocalDate localDate, int monthsToSubtract) {
+        checkPositive(monthsToSubtract, "monthsToSubtract");
+        return changeMonth(localDate, -monthsToSubtract);
+    }
+
+    /**
+     * Obtains a copy of this local date with the specified number of days subtracted.
+     */
+    public static LocalDate minusDays(LocalDate localDate, int daysToSubtract) {
+        checkPositive(daysToSubtract, "daysToSubtract");
+        return changeDays(localDate, -daysToSubtract);
+    }
+
+    private static LocalDate changeYear(LocalDate localDate, int yearsToChange) {
```

Comment:
Please document this method. Describe that `yearsToChange` can be positive and negative. I would suggest renaming the last parameter to `yearsDelta` (you're not changing these years, it's the difference you want to apply). After renaming not having a zero value would be more obvious. But please do have precondition for this.